byteps/tensorflow/__init__.py

```diff
@@ -16,26 +16,35 @@
 # ==============================================================================
 # pylint: disable=g-short-docstring-punctuation
 
-from __future__ import absolute_import
```

Comment:
Why do we remove these __future__ imports? I don't think we are dropping support for python2

Reply:
I will add it back.